test/getGrist.ts

```diff
@@ -129,7 +134,9 @@ export class GristUtils extends GristWebDriverUtils {
       try {
         const url = this.url;
         const resp = await fetch(url + '/status');
-        if (resp.status === 200) { break; }
+        if (resp.status === 200) {
```

Comment:
All these small changes mean a reviewer has to squint to see if there are meaningful changes - although whitespace changes could be turned off I guess.  I'd encourage you going forward to trim your PRs to necessary changes.